android/app/build.gradle

```diff
@@ -39,6 +39,7 @@ android {
         versionCode flutterVersionCode.toInteger()
         versionName flutterVersionName
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        multiDexEnabled true
```

Comment:
multidex needs to be enabled as per https://developer.android.com/studio/build/multidex